MdePkg/MdePkg.dsc

```diff
@@ -34,6 +34,7 @@
 
 [LibraryClasses]
   SafeIntLib|MdePkg/Library/BaseSafeIntLib/BaseSafeIntLib.inf
+  TimerLib|MdePkg/Library/BaseTimerLibNullTemplate/BaseTimerLibNullTemplate.inf

```

Comment:
How is this related to these changes?  I do not see any new usage of TimerLib

Reply:
Hello Michael,
the patch is actually not needed, it solved a dependency I introduced in a local patch ...